scrapers/DorcelVision.yml

```
@@ -10,34 +10,65 @@ movieByURL:
     url:
       - https://www.dorcelvision.com/en/movies
     scraper: movieScraper
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://www.dorcelvision.com/en/search?type=4&keyword={}
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
 
 xPathScrapers:
+  sceneSearch:
+    common:
+      $card: //div[@id="entriesList"]/div
+    scene:
+      Title: $card/details/div/h3
+      URL: 
+        selector: $card/a[@class="movies rolloverv2"]/@href
+        postProcess: 
+          - replace:
+            - regex: (.*)
+              with: https://www.dorcelvision.com$1
+              
+      Image: 
+        selector: $card/a[@class="movies rolloverv2"]/img[last()]/@data-src
+        postProcess: 
+          - replace:
+              - regex: (.*)
+                with: https://www.dorcelvision.com$1
+      Details: $card/details/summary
+        
```

Let's grab the performers while we're at it 👍 
```suggestion
      Performers:
        Name:
          selector: $card/details//p[@class="persons"]
          postProcess:
            - replace:
                - regex: ^With:\s+
                  with:
                - regex: \s*,\s*
                  with: __SEPARATOR__
          split: __SEPARATOR__
```

---

I couldn't manage to make this work when I tried, it returns always the same list of performers for every result, even ones where is no performers listed.

---

I apologize, I thought I tested this but upon retrying it now I can see the exact issue you're describing: disregard this comment 😅